.github/component_owners.yml

```diff
@@ -169,7 +169,7 @@ components:
   test/OpenTelemetry.ResourceDetectors.Process.Tests/:
     - Kielek
     - lachmatt
-  test/OpenTelemetry.ResourceDetectors.ProcessRuntime.Tests/:
+  test/OpenTelemetry.Resources.ProcessRuntime.Tests/:
```

Comment:
Missed in previous PR. Could you please ensure that we have alphabetical order across the file?